### PR TITLE
CANS-329: Change Tab behavior on Expand Question Details

### PR DESCRIPTION
**Pull request by zliuwd**
## Description

https://osi-cwds.atlassian.net/browse/CANS-329
When the User tabs through the assessment, the Question Details (+ sign to the left of the Question Name) DO NOT expand when tabbing past the + sign.
When the User has focused on the + sign (by tabbing through the assessment), they can expand the Question details by pressing spacebar
The user can still click the + sign to expand
## Tests

- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/Item.js [hunk 1]
```diff
@@ -39,6 +39,14 @@ class Item extends Component {
     });
   }
 
+  handleKeyCheck = e => {
+    if (e.keyCode !== 9 || e.which !== 9) {
```
Comment (nasar-cambria) [inline app/javascript/Application/components/Assessment/Item.js]: As discussed ```if (e.which !== 9)``` this should work just fine. e.which property normalizes e.keycode

File: app/javascript/Application/components/Assessment/Item.js [hunk 2]
```diff
@@ -39,6 +39,14 @@ class Item extends Component {
     });
   }
 
+  handleKeyCheck = e => {
```
Comment (edgibbs) [inline app/javascript/Application/components/Assessment/Item.js]: I'm assume e is an `event`?

File: app/javascript/Application/components/Assessment/Item.js (hunk 1 above)
Comment (edgibbs) [inline app/javascript/Application/components/Assessment/Item.js]: 9 is a magic number should probably just make it a `TAB` constant.

File: app/javascript/Application/components/Assessment/Item.js [hunk 3]
```diff
@@ -39,6 +39,14 @@ class Item extends Component {
...
+    if (e.which !== 9) {
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/Item.js]: The attribute `which` is deprecated; you should use `key` instead, if available.

File: app/javascript/Application/components/Assessment/Item.js [hunk 4]
```diff
@@ -183,7 +191,9 @@ class Item extends Component {
               tabIndex={0}
               className={classes}
               onClick={this.switchExpandedState}
-              onKeyDown={this.switchExpandedState}
+              onKeyDown={e => {
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/Item.js]: Try to replace the line with 
`onKeyDown={this.handleKeyCheck}`
not to create a new function

**Pull request closed without merging**